internal/provider/resource_directory_role_collection_assignment.go

```diff
@@ -120,6 +120,9 @@ func (rs *directoryRoleCollectionAssignmentResource) Read(ctx context.Context, r
 		return
 	}
 
+	// Setting ID of state - required by hashicorps terraform plugin testing framework for Import . See issue https://github.com/hashicorp/terraform-plugin-testing/issues/84
+	state.Id = state.RoleCollectionName
```

Comment:
Same like with the other PRs. It's only unique if we set it to: `<subaccountid>,<rolecollection>,<user>`

Reply:
Fixed